utensor_cgen/backend/utensor/code_generator/rearch/_code_generator.py

```diff
@@ -83,33 +91,60 @@ def _find_required_ops(self, ugraph):
         )
     return ops, placeholders, tensor_var_map
 
-  def _try_get_opt_snippets(self, ugraph, ops, tensor_var_map):
+  def _try_get_opt_snippets(self, ugraph, ops, tensor_var_map, placeholders):
+    # snippets for declaring ops
+    declare_op_snippets = []
+    construct_op_snippets = []
     # snippets to be rendered in the generated function body
     local_snippets = []
     # snippets to be rendered in the weight header file
     weight_snippets = []
+    # inputs/outputs enum names
+    input_enums = []
+    output_enums = []
     # op -> op variable name
     ops_map = {}
     for i, op in enumerate(ops):
       op_var_name = 'op_{:03d}'.format(i)
```

Comment:
How hard would it be to make this `op_var_name = 'op_{:03d}_{}'.format(i, op_type.lower())`?  It would be nice to make the op vars more explicit since they are now in a different file

Reply:
Since ops of same op_type but different template parameters would have the same variable name.

Reply:
yup yup, This proposal would still have the same numbering but just append the op_type
so `op_001, op_002, op_003` would become `op_001_conv2d, op_002_maxpool, op_003_conv2d`f

Reply:
Oh, I see.
Easy fix then.

Reply:
Done.
Now is named by `'op_{}_{:03d}'.format(op.op_type, i)`